app/actualize_script.php

```diff
@@ -39,7 +38,11 @@ function notice($message) {
 notice('FreshRSS starting feeds actualization at ' . $begin_date->format('c'));
 
 // make sure the PHP setup of the CLI environment is compatible with FreshRSS as well
+echo 'Failed requirements!', "\n";
```

Comment:
This is in order to also have a message on STDOUT for this specific script, in addition of STDERR (which contain all messages), which is apparently not seen by all users.